src/main.js

```diff
@@ -63,21 +82,11 @@ function displayRandomHomePage() {
   displayRandomTagline1();
   displayRandomTagline2();
 }
-
-
-//
-// function createRandomHomePage() {
-//   var cover = covers[getRandomIndex(covers)];
-//   var title = titles[getRandomIndex(titles)];
-//   var tagline1 = descriptors[getRandomIndex(descriptors)];
-//   var tagline2 = descriptors[getRandomIndex(descriptors)];
-//   currentCover = new Cover(cover, title, tagline1, tagline2);
-//   displayRandomHomePage();
-// }
-//
-// function displayRandomHomePage() {
-//   htmlImage.src = currentCover.coverImgSrc;
-//   htmlTitle.innerText = currentCover.title;
-//   htmlTagline1.innerText = currentCover.descriptor1;
-//   htmlTagline2.innerText = currentCover.descriptor2;
-// }
+//ITERATION 2 FORM PAGE & MENU UPDATE
```

Comment:
I'm going to work on making a toggle function that houses all of the hidden classes!